### DISCUSSION TITLE
Packages section on profile is reporting incorrect packages count

### DISCUSSION THREAD

**Discussion by hwittenborn** [community/community#7930] [Actions]
If you look at the [`Packages` section on my profile](/hwittenborn?tab=packages), it reports I have one package under my account, even though I have zero.

I deleted the one package the other day, so I'm assuming something funky happened after the package got deleted.

**Comment by ghost** [comment]
I've got the same issue with a few of projects, where the number of package dangling around after removal of packages.

May I know which kind of registry are you using (for me, it's NuGet)? I'd like to know if the problem exists in only certain type or multiple of them...

**Comment by steffbeckers** [comment]
I got the same issue. Already tried to query the GET /user/packages?package_type=...  API endpoint with all possible package types, but no results.

![image](https://github.com/community/community/assets/7551394/96e201b4-eb3f-4428-8b3d-7ad2ab3acaa2)

![image](https://github.com/community/community/assets/7551394/8f39de75-34db-4ec2-ab32-2d617bc9e96a)
